.travis.yml

```diff
@@ -1,23 +1,24 @@
 language: python
 python:
- - '2.6'
  - '2.7'
  - '3.4'
  - '3.5'
+ - '3.6'
 install:
- - python setup.py install
- - pip install codecov
+  - make install
+  - make test-install
+  - pip install codecov
 script:
- - python test/__init__.py
- - coverage run test/__init__.py
+  - make test
+  - . venv/bin/activate; coverage run test/__init__.py
 after_success:
- - codecov
-notifications:
-  hipchat:
-    rooms:
-      secure: XbJIz0PtZo+/G6UCnCxLgYDPk7OS8fpyiQwpnfGBr2xW1Z5vhWS+jrR6GndAFJfW9mmgcvq9dJL4bSThITebcZJJtztfuWnGUnONKbfSsFcuEr6o3uGPNmfVwZ9JAL78BUU861A/ykcWe/6lkovlUphkDLa2/l1r7t7TGD75f5k=
-    template:
-    - '<a href="https://travis-ci.org/%{repository}/builds/%{build_id}">%{repository}
-      Build %{build_number}</a> on branch <i>%{branch}</i> by %{author}: <strong>%{message}</strong>
-      <a href="https://github.com/%{repository}/commits/%{commit}">View on GitHub</a>'
-    format: html
+  - codecov
+deploy:
+  provider: pypi
+  user: __token__
+  password:
+    secure: $PYPI_TOKEN
```

Comment:
Thought we were going the other way and not using env vars.

Reply:
I tried to encrypt the token in this repo and was getting a travis CLI error about the data being too large to encrypt (which is weird, because it worked for python-http-client)